design/admin/templates/ajaxuploader/location.tpl

```diff
@@ -1,8 +1,38 @@
 {* DO NOT EDIT THIS FILE! Use an override template instead. *}
 <form action={'ezjscore/call'|ezurl} method="post" class="ajaxuploader-location">
     <fieldset>
-        <legend>{"Step 2/3: Choose a location for the new '%class' object"|i18n( 'design/admin/ajaxuploader', '', hash( '%class', $class.name|wash() ) )}</legend>
+        {def
+            $upload_class_map        = ezini( 'CreateSettings', 'MimeClassMap', 'upload.ini' )
+            $upload_class_attribute  = ezini( concat( $upload_class_map['image'], '_ClassSettings' ), 'FileAttribute', 'upload.ini' )
```

Comment:
Technically we might want to check that it begins with image, since people can specify specific MIME types such as image/jpeg. But we can save that for a future improvement.